sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceClientBuilder.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.annotation.ServiceClientBuilder;
+
+/**
+ * builds the table service clients
+ */
+@ServiceClientBuilder(serviceClients = {TableServiceClient.class, TableServiceAsyncClient.class})
+public class TableServiceClientBuilder {
```

Comment:
HTTP service builders should have all these methods:
https://azure.github.io/azure-sdk/java_design.html#java-service-client-builder-consistency